code/modules/client/client_procs.dm

```diff
@@ -576,7 +576,7 @@ GLOBAL_LIST_INIT(blacklisted_builds, list(
 	if (src.holder && src.holder.rank)
 		admin_rank = src.holder.rank.name
 	else
-		if (!GLOB.deadmins[ckey] && check_randomizer(connectiontopic))
+		if (!GLOB.deadmins[ckey])
```

Comment:
This change doesn't do what you think it does

Reply:
To help you a step forward goof:

Old: If they are not in the deadmin list AND they appear to be using a randomizer, we early return.
This condition only passes if they're using a CID randomizer and are not in the the deadmin list.

New: If they are not in the deadmin list, we early return.
This conidition will always pass if the player is not in the deadmin list.

The logic of your new code is totally wrong. It's gone from almost never being true, to almost always being true.